tests/playwright/controls.py

```diff
@@ -2909,14 +2982,87 @@ def expect_header(
     #     )
 
     def expect_max_height(self, value: StyleValue, *, timeout: Timeout = None) -> None:
+        """
+        Expects the card to have a specific maximum height.
+
+        Parameters
+        ----------
+        value
+            The expected maximum height value.
+        timeout
+            The maximum time to wait for the expectation to pass. Defaults to None.
+        """
         expect_to_have_style(self.loc_container, "max-height", value, timeout=timeout)
 
     def expect_min_height(self, value: StyleValue, *, timeout: Timeout = None) -> None:
+        """
+        Expects the card to have a specific minimum height.
+
+        Parameters
+        ----------
+        value
+            The expected minimum height value.
+        timeout
+            The maximum time to wait for the expectation to pass. Defaults to None.
+        """
         expect_to_have_style(self.loc_container, "min-height", value, timeout=timeout)
 
     def expect_height(self, value: StyleValue, *, timeout: Timeout = None) -> None:
+        """
+        Expects the card to have a specific height.
+
+        Parameters
+        ----------
+        value
+            The expected height value.
+        timeout
+            The maximum time to wait for the expectation to pass. Defaults to None.
+        """
         expect_to_have_style(self.loc_container, "height", value, timeout=timeout)
 
+    def expect_full_screen_available(
```

Comment:
Double check if we can remove the method from valuebox as it is a card under the hood. If it is not a card, leave as is.